dags/covid_etl_dag.py

```diff
@@ -132,12 +172,15 @@ def create_dynamic_etl(task_id, callable_function):
         upload_to_s3_task = create_dynamic_etl('upload_to_s3', upload_to_s3)
 
         upload_to_snowflake_task = upload_to_snowflake('upload_to_snowflake')
+        
+        run_qa_task = PythonOperator(task_id="run_qa", python_callable=qa_checks)
 
         start >> cleanup_output_folder_task
         cleanup_output_folder_task >> execute_notebook_task
         execute_notebook_task >> upload_to_s3_task
         upload_to_s3_task >> upload_to_snowflake_task
-        upload_to_snowflake_task >> end
+        upload_to_snowflake_task >> run_qa_task
```

Comment:
Question: should we only include this task if there is a QA SQL in the matching folder?

Reply:
We could, but I plan for every notebook to have QA, if nothing else to make sure the final table has data.